test/setup/main.tf

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "project" {
+  source  = "terraform-google-modules/project-factory/google"
+  version = "~> 14.0"
+
+  name              = "ci-gen-ai-document-summarization"
+  random_project_id = "true"
+  org_id            = var.org_id
+  folder_id         = var.folder_id
+  billing_account   = var.billing_account
+
+  activate_apis = [
```

Comment:
question: do we need to activate Vertex AI, Vision, and BigQuery here?

Reply:
These tests have actually never been run, because we haven't hooked up the CFT infrastructure. Lets hold off on adding them here, and wait to see if they are needed (they _are_ added in the main.tf for the solution)